From 811dd28b9e87933c6c3a9dbd6ebcc6f8d5d0c0d9 Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Wed, 4 Dec 2019 18:32:59 +0100
Subject: [PATCH 17/18] Revert "Makefile: add -bfq version suffix"

Makes it difficult to apply to kernel 5.4.1 and later.

This reverts commit 8b20b8087d330fd882c5f2d12916a3b6a6590ad2.

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index e9d95b2f8e89..d4d36c61940b 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 5
 PATCHLEVEL = 4
 SUBLEVEL = 0
-EXTRAVERSION = -bfq
+EXTRAVERSION =
 NAME = Kleptomaniac Octopus
 
 # *DOCUMENTATION*
-- 
2.25.0.114.g5b0ca878e0.dirty

